app/src/main/res/layout/onboarding_firefox_account.xml

```diff
@@ -51,40 +47,28 @@
         android:backgroundTint="@color/onboarding_card_button_background_dark"
         android:clickable="true"
         android:focusable="true"
-        android:foreground="?android:attr/selectableItemBackground"
+        android:gravity="center"
         android:padding="10dp"
-        android:visibility="gone">
-        <TextView
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:layout_gravity="center"
-            android:clickable="false"
-            android:focusable="false"
-            android:textStyle="bold"
-            android:gravity="center"
-            android:text="@string/onboarding_firefox_account_stay_signed_in"
-            android:textColor="?neutral"
-            android:textSize="14sp" />
-    </FrameLayout>
-    <FrameLayout
+        android:text="@string/onboarding_firefox_account_stay_signed_in"
+        android:textAllCaps="false"
+        android:textColor="?neutral"
+        android:textSize="14sp"
+        android:textStyle="bold"
+        android:visibility="gone" />
+
+    <Button
         android:id="@+id/sign_out_button"
```

Comment:
@boek are these three buttons supposed to look the same? The last one didn't have a background so I added it.

Reply:
No, the sign out button has no background

Reply:
We're not shipping this right now, so I'm going to go ahead and land this now